cli/internal/plugins/bzr/bzr.go

```diff
@@ -12,41 +12,70 @@ import (
 	"github.com/gobuffalo/buffalo-cli/plugins/plugprint"
 )
 
-type Buffalo struct{}
+const (
+	name        = "bzr"
+	description = "Provides bzr related hooks to Buffalo applications."
+)
+
+//Ensuring bzr is a describer
+var _ plugprint.Describer = Buffalo{}
 
+//Ensuring bzr is a buffalo Plugin
+var _ plugins.Plugin = Buffalo{}
+
+//Ensuring bzr is a buffalo buildcmd.Versioner
 var _ buildcmd.Versioner = &Buffalo{}
 
-func (b *Buffalo) BuildVersion(ctx context.Context, root string) (string, error) {
-	if _, err := exec.LookPath("bzr"); err != nil {
-		return "", err
-	}
+//testConfig is used to mock command things when testing.
+var testConfig = struct {
+	enabled bool
 
-	bb := &bytes.Buffer{}
+	resultError   error
+	resultVersion string
+}{}
+
+type Buffalo struct{}
+
+func (b *Buffalo) runCmd(ctx context.Context, bb *bytes.Buffer) error {
+	if testConfig.enabled {
```

Comment:
Try using an interface here, like the one from `buffalo build`

```go
type Runner interface {
	plugins.Plugin
	RunBuild(ctx context.Context, cmd *exec.Cmd) error
}
```

https://github.com/gobuffalo/buffalo-cli/blob/resource/cli/internal/plugins/buildcmd/ifaces.go#L64
https://github.com/gobuffalo/buffalo-cli/blob/resource/cli/internal/plugins/buildcmd/ifaces_test.go#L213
https://github.com/gobuffalo/buffalo-cli/blob/resource/cli/internal/plugins/buildcmd/main_test.go#L14